src/main.js

```diff
@@ -114,8 +121,11 @@ const bot = new Client({
 
 bot.on("ready", async () => {
   // when loaded (ready event)
-  bot.user.setActivity(`${PREFIX}help | ${PREFIX}info`, { type: "PLAYING" });
+  bot.user.setActivity(configManager.config.activity.description ?? "nil", {
+    type: configManager.config.activity.type.toUpperCase() ?? "PLAYING",
+  });
   utils.logger.log("debug", `${bot.user.username} is ready...`);
+  console.info(`${bot.user.username} is ready...`);
```

Comment:
Making an exception to the "No `console.log`" style guide rule for this instance. Regardless of the console log level, there should be some feedback in the console that the bot has started.